mycity/mycity/test/unit_tests/test_get_alerts_intent.py

```diff
@@ -67,3 +67,38 @@ def test_that_prune_normal_responses_returns_an_empty_dictionary_if_all_response
         alerts[get_alerts_intent.Services.TRASH.value] = 'Too normal in my opinion'
         alerts[get_alerts_intent.Services.TOW_LOT.value] = get_alerts_intent.TOW_LOT_NORMAL_MESSAGE
         self.assertEqual({}, get_alerts_intent.prune_normal_responses(alerts))
+
+    def test_inclement_weather_alert_returns_normal_response_for_no_alert(self):
+        response = get_alerts_intent.get_inclement_weather_alert(
+            self.request,
+            self.stub_get_alerts)
+        self.assertEqual(constants.NO_INCLEMENT_WEATHER_ALERTS, response.output_speech)
+
+    def test_inclement_weather_alert_returns_normal_response_for_keyword_in_wrong_alert_section(self):
+        self.get_alerts_stub_return_dictionary = {
+            get_alerts_intent.Services.PARKING_METERS: "Snow doesn't bother us!"
+        }
+        response = get_alerts_intent.get_inclement_weather_alert(
```

Comment:
Nitpick: Do these function calls need to be split across three lines? Does PEP complain? If so, ignore

Reply:
They do not, this is habit from code style guidelines used at iZotope. I'll make it one line.